WordProject/src/wordCount/Main.java

```diff
@@ -0,0 +1,72 @@
+
+		String newText=oldText.toLowerCase();
+
+		String unText=newText.replaceAll("[[\\.\\?\\!\\,\\;\\:\\{\\}\\(\\)\\']]", "");
+
+		Scanner input=new Scanner(unText);
+
+		ArrayList<String> words = new ArrayList<String>();
+
+		ArrayList<Integer> count= new ArrayList<Integer>();
+
+		while(input.hasNext())
+		{
+			//get next
+			String nextWord = input.next();
+			// if word is in ArrayList
+			if(words.contains(nextWord))
+			{
+				int index = words.indexOf(nextWord);
+				count.set(index, count.get(index)+1);
+			}
+			else
+			{
+				words.add(nextWord);
+				count.add(1);
+			}
+
+		}
+
+		input.close();
+		//print result
+
+		for(int i=0; i<words.size();i++)
+		{
+			// Collections.sort(words);
+			// Collections.reverse(words);
+			Collections.sort(count);
+			Collections.reverse(count);
```

Comment:
You don't need to sort and reverse inside the loop - since these methods mutate the argument passed into them, just use them once outside the `for` loop.

On another note, keep in mind that sorting `words` and `count` as separate lists will not make the indices correspond to the same word and frequency count anymore. **The results are correct before either of the lists are sorted**, but not after. You will need to find a way to keep each word connected to its corresponding frequency count (I would suggest using class instances or `HashMap` types for this).